BASE/src/Microsoft.ApplicationInsights/Channel/Transmission.cs

```diff
@@ -209,12 +214,24 @@ public virtual async Task<HttpWebResponseWrapper> SendAsync()
                         }
                     }
                     catch (OperationCanceledException)
-                    {                        
+                    {
                         wrapper = new HttpWebResponseWrapper
                         {
                             StatusCode = (int)HttpStatusCode.RequestTimeout,
                         };
                     }
+                    finally
+                    {
+                        try
+                        {
+                            // Initiates event notification to subscriber with Transmission and TransmissionStatusEventArgs.
+                            this.TransmissionStatusEvent?.Invoke(this, new TransmissionStatusEventArgs(wrapper ?? new HttpWebResponseWrapper() { StatusCode = 0 }));
```

Comment:
we should use the same status code we use for logging.. I think we use 999 or something?

Reply:
Yes. It is [999](https://github.com/microsoft/ApplicationInsights-dotnet/blob/7633ae849edc826a8547745b6bf9f3174715d4bd/BASE/src/ServerTelemetryChannel/Implementation/ResponseStatusCodes.cs#L8).